frontend/src/pages/premium/waitlist.test.tsx

```diff
@@ -1,16 +1,54 @@
 import React from "react";
-import { act, render } from "@testing-library/react";
+import { act, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import { axe } from "jest-axe";
+import { type Props as CountryPickerProps } from "../../components/waitlist/countryPicker";
+import { type Props as LocalePickerProps } from "../../components/waitlist/localePicker";
 import { mockFluentReact } from "../../../__mocks__/modules/fluent__react";
 import { mockNextRouter } from "../../../__mocks__/modules/next__router";
 import { mockConfigModule } from "../../../__mocks__/configMock";
+import { setMockRuntimeData } from "../../../__mocks__/hooks/api/runtimeData";
+import { setMockUserData } from "../../../__mocks__/hooks/api/user";
+import { setMockProfileData } from "../../../__mocks__/hooks/api/profile";
 
 import PremiumWaitlist from "./waitlist.page";
 
 jest.mock("@fluent/react", () => mockFluentReact);
 jest.mock("next/router", () => mockNextRouter);
 jest.mock("../../config.ts", () => mockConfigModule);
 jest.mock("../../hooks/gaViewPing.ts");
+jest.mock("../../components/waitlist/countryPicker.tsx", () => ({
+  // We're mocking out the country picker because it dynamically imports the
+  // list of available countries, which would make the test pretty cumbersome
```

Comment:
question (non-blocking): what does "pretty cumbersome" mean in this context? Lots of extra testing code? Slower testing runs? Both?

Reply:
Tried to clarify that in 12a89486.